sites/public/__tests__/components/listing/listing_sections/Neighborhood.test.tsx

```diff
@@ -21,10 +21,45 @@ describe("<Neighborhood>", () => {
           longitude: 2,
         }}
         name={"Listing name"}
+        neighborhood={"Westend"}
+        region={"Downtown"}
+        neighborhoodAmenities={{ groceryStores: "Market", pharmacies: "Health store" }}
       />
     )
-    expect(getAllByText("Neighborhood").length).toBeGreaterThan(0)
-    expect(getAllByText("Location and transportation").length).toBeGreaterThan(0)
-    expect(getByText("Get Directions")).toBeDefined()
+    expect(screen.getAllByText("Neighborhood").length).toBe(3)
+    expect(screen.getAllByText("Location and transportation").length).toBe(2)
+    expect(screen.getByText("Get Directions")).toBeDefined()
+    expect(screen.getByText("Westend")).toBeDefined()
+    expect(screen.getByText("Region")).toBeDefined()
+    expect(screen.getByText("Downtown")).toBeDefined()
+    expect(screen.getByText("Within 2 miles")).toBeDefined()
+    expect(screen.getByText("Grocery Stores")).toBeDefined()
+    expect(screen.getByText("Market")).toBeDefined()
+    expect(screen.getByText("Pharmacies")).toBeDefined()
+    expect(screen.getByText("Health store")).toBeDefined()
```

Comment:
chore: It is [recommended](https://testing-library.com/docs/queries/about/#priority) to use getByRole whenever possible. This was we can test against the accessibility tree. So for this section we can get the heading and links using the getByRole but will still need to do the getByText for the values under the heading

examples for this file:
```
screen.getByRole("link", { name: "Get Directions (opens in a new tab)" })
```
and 
```
screen.getByRole("header", { name: "Grocery Stores" })
```

Reply:
@ludtkemorgan I tried this, but all of the names were coming through empty - I will dig in more!

Reply:
Yeah, I can add it to the eng workshop. Sorry about the wrong example. It should have been "heading" instead of "header", but looks like you figured that out. Thanks for making those changes!